src/aur.rs

```diff
@@ -25,7 +44,6 @@ impl FromStr for AurHelper {
                     String::from("--useask"),
                     String::from("--removemake"),
                     String::from("--norebuild"),
-                    String::from("--noconfirm"),
```

Comment:
Why remove this?

Reply:
It is included twice currently and only  needed once
https://github.com/r-darwish/alma/blob/6b49b5f262e2cd1fd66ca56ad2024417c5ddf886/src/aur.rs#L28-L34